freeciv-web/src/main/webapp/webclient/index.jsp

```diff
@@ -6,45 +6,43 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%
 String googleSigninClientKey = null;
-String fcwHost = null;
 String trackJsToken = null;
+boolean fcwDebug = false;
+String fcwMinified = "";
 try {
   Properties prop = new Properties();
   prop.load(getServletContext().getResourceAsStream("/WEB-INF/config.properties"));
   googleSigninClientKey = stripToEmpty(prop.getProperty("google-signin-client-key"));
   trackJsToken = stripToNull(prop.getProperty("trackjs-token"));
-  fcwHost = prop.getProperty("fcw_host");
+
+  String debugParam = request.getParameter("debug");
+  fcwDebug = (debugParam != null && (debugParam.isEmpty() || parseBoolean(debugParam)));
+  fcwMinified = fcwDebug ? ".min" : "";
```

Comment:
You surely meant the opposite: debug -> raw, !debug -> min.

Reply:
That is a distinct possibility :smirk:.

Fixed.